warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexCachingIteratorJexl.java

```diff
@@ -1427,7 +1476,9 @@ protected String getOwnerId(Object owner) {
         
         public void takeOwnership(String row, Object owner) throws IOException {
             Path file = getOwnershipFile(row);
+            log.warn("Creating ownership file for " + controlDir);
             writeFile(file, getOwnerId(owner).getBytes());
+            log.warn("Created ownership file for " + controlDir);
```

Comment:
Do we have a sense for how noisy these log statements will be?

Reply:
yea, I should probably remove these